.gitignore

```diff
@@ -257,6 +257,7 @@ Makefile.am-stamp
 /winbuild
 /man
 /README
+/build
```

Comment:
remove this change, and many non essetial changes (only relevant in your personal environment) should be removed